usefulUtilities/sitV2/appResources/appData/resources/sittableUIClient.js

```diff
@@ -92,7 +93,7 @@
             if (DEBUG) {
                 console.log("MOUSE RELEASE");
             }
-            if (event.isPrimaryButton) {
+            if (event.isPrimaryButton && Vec3.distance(MyAvatar.position,Entities.getEntityProperties(_this.entityID,["position"]).position) <= MAX_SIT_DISTANCE_M) {
```

Comment:
please split up to multiple lines with proper indentation (as per my earlier suggestion)